modules/programs/neomutt.nix

```
@@ -196,7 +196,16 @@ let
 
   accountStr = account:
     with account;
-    ''
+    let
+      signature = (if (account.signature.showSignature == "none") then
+        "unset signature"
+      else if account.signature.command != null then
+        ''set signature = "${account.signature.command}|"''
```

```suggestion
        ''set signature = "${toString account.signature.command}|"''
```
If my prior suggestion is applied, it becomes necessary.